.agent/commands/pr-bots-review.md

```
@@ -31,12 +31,15 @@ HEAD_SHA="$(git rev-parse HEAD)"
 echo "$HEAD_SHA"
 ```
 
-2. Comment `@codex review` on the PR (include the head SHA so the bot reviews the current PR state).
-The comment body should be Japanese, but keep `@codex review` exactly as-is.
+2. Resolve review mention from `AGENTIC_SDD_PR_REVIEW_MENTION` and comment it on the PR
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)